drivers/src/sha2_512_384.rs

```diff
@@ -4,50 +4,65 @@ Licensed under the Apache-2.0 license.
 
 File Name:
 
-    sha384.rs
+    sha2_512_384.rs
 
 Abstract:
 
-    File contains API for SHA-384 Cryptography operations
+    File contains API for SHA2-512/384 Cryptography operations
 
 --*/
 
 use core::usize;
 
 use crate::kv_access::{KvAccess, KvAccessErr};
 use crate::PcrId;
-use crate::{array::Array4x32, wait, Array4x12, Array4x8};
+use crate::{array::Array4x32, wait, Array4x12, Array4x16, Array4x8};
 #[cfg(not(feature = "no-cfi"))]
 use caliptra_cfi_derive::cfi_impl_fn;
 use caliptra_error::{CaliptraError, CaliptraResult};
 use caliptra_registers::sha512::Sha512Reg;
 
-const SHA384_BLOCK_BYTE_SIZE: usize = 128;
-const SHA384_BLOCK_LEN_OFFSET: usize = 112;
-const SHA384_MAX_DATA_SIZE: usize = 1024 * 1024;
+const SHA2_BLOCK_BYTE_SIZE: usize = 128;
+const SHA2_BLOCK_LEN_OFFSET: usize = 112;
+const SHA2_MAX_DATA_SIZE: usize = 1024 * 1024;
```

Comment:
naming nit: Different SHA2 modes has different block sizes, it's just that 384 and 512 have the same block size. Similar to but other comment, I would probably just name these as `SHA512_*`

Reply:
Updated.